sites/public/src/components/shared/CustomIconMap.tsx

```diff
@@ -0,0 +1,62 @@
+import { t } from "@bloom-housing/ui-components"
+
+export const CustomIconMap = {
```

Comment:
More of a nit, but we could consider exporting a type from this file of all the icons in here (for now just application, profile) and then use that type in AccountCard, so the prop is more generically "a custom icon".

Reply:
+1 on this. Also curious if you ever anticipate wanting each of the icons as separate exports? The way it is now is that you will always have to do `CustomIconMap["application"]` rather than just pulling in the application icon. It works for our current use case, but wonder if it limits us down the road.